dandischema/models.py

```diff
@@ -423,6 +423,12 @@ class Contributor(DandiBaseModel):
         description="Identifier associated with a sponsored or gift award.",
         nskey="dandi",
     )
+    orcidID: Optional[Identifier] = Field(
+        None,
+        title="Orcid ID",
+        description="Orcid ID....",
+        nskey="dandi",
+    )
```

Comment:
we already have orcids as `identifier` for `Person` class.

Reply:
forgot to ask, assumed that you want to have a new field only for ORCID. However `identifier` is also used for ORCID only right now... 
Will change